api/pom.xml

```diff
@@ -79,6 +79,30 @@
             <properties>
                 <quarkus.package.type>native</quarkus.package.type>
             </properties>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <version>${surefire-plugin.version}</version>
```

Comment:
usually the failsafe and surefire plugins are released together. it is used this way inQuarkus doc and all Quarkus examples